src/common/SPELLS/classic/warlock.ts

```diff
@@ -273,7 +283,7 @@ const spells = {
   },
   SUMMON_INFERNO: {
     id: 1122,
-    name: 'Inferno',
+    name: 'Summon Inferno',
     icon: 'spell_shadow_summoninfernal.jpg',
   },
```

Comment:
this is "Summon Infernal", right? https://www.wowhead.com/cata/spell=1122/summon-infernal